kmod/src/dir.c

```diff
@@ -1615,8 +1615,9 @@ static int verify_ancestors(struct super_block *sb, u64 p1, u64 p2,
  * from using parent/child locking orders as two groups can have both
  * parent and child relationships to each other.
  */
-static int scoutfs_rename(struct inode *old_dir, struct dentry *old_dentry,
-			  struct inode *new_dir, struct dentry *new_dentry)
+static int scoutfs_rename_common(struct inode *old_dir,
+			  struct dentry *old_dentry, struct inode *new_dir,
```

Comment:
I know it's tedious, but I use tabs+spaces to line up the additional lines of arguments with the first arguments.

Reply:
I normally would have this right... But stupid editor i use doesnt display it right... VS code on the mac. Dunno why; i havent really looked into it yet since when my laptop comes in ill be switching anyways :/